### PR TITLE
Birth by Sleep documentation.

### PR DESCRIPTION
**Pull request by Truthkey**
This includes updated documentation.

Updated information:

- PMO
- PMP
- PAM

New file formats:
- EXA
- EPD
- OLO
- BEP
- PTX

It also includes changes to some BBS constants, an update to BBSA's hash list and a fix for Tm2's class.

### CODE REVIEW

File: docs/bbs/bep.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# BEP Format
```
Comment (Topaz-Reality) [inline docs/bbs/bep.md:1]: Can you add a linefeed after all of the titles? GitHub is known to break formatting when titles are present without a linefeed.

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]
Looking good to me so far, just one formatting change is necessary.

File: docs/bbs/bep.md (hunk 1 above)
Comment (Truthkey) [reply docs/bbs/bep.md:1]: Alright, seems fair. I already made the changes.

**Comment by Topaz-Reality** [comment]
Waiting for Azure Pipelines.

**Review by Topaz-Reality — APPROVED** [review]
Seems all good to me.

**Pull request merged**